hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/BlockStreamBuilder.java

```diff
@@ -1135,6 +1137,13 @@ private void addOutputItemsTo(@NonNull final List<BlockItem> items) {
             items.add(itemWith(
                     TransactionOutput.newBuilder().submitMessage(new SubmitMessageOutput(assessedCustomFees))));
         }
+
+        if (functionality == CRYPTO_CREATE && accountId != null) {
```

Comment:
Since this is added, we can change the `CryptoCreateTranslator` to look at the `CryptoCreateOutput` instead of StateChanges.